exampleSite/content/blog/latest/index.md

```diff
@@ -0,0 +1,11 @@
++++
+date = "2018-06-05"
+fragment = "blog"
+weight = 50
+
+#background = ""
+title = "Latest articles"
+
+article_count = 5
+blog_directory = "/blog"
```

Comment:
This should probably be a category or tag instead of a directory. Feels inconsistent with the usual way blog in Hugo work.

Reply:
At first I was trying to implement the fragment in a way that it could list any page in any section but later forgot about this variable as I changed my mind. I'll add support for category and tags.

Reply:
You can take a look at how the news section (list to content) is structure on the gohugo site.
https://github.com/gohugoio/hugoDocs/tree/master/themes/gohugoioTheme/layouts/news

So I would say one could provide a category or tag. These would then be used to look up the actual pages/articles.
Mixing tags and categories seems unnecessary and more complex to implement.